packages/exec/src/interfaces.ts

```diff
@@ -30,6 +30,9 @@ export interface ExecOptions {
   /** optional. How long in ms to wait for STDIO streams to close after the exit event of the process before terminating. defaults to 10000 */
   delay?: number
 
+  /** optional. input to write to the process on STDIN. */
+  stdin?: Buffer
```

Comment:
https://nodejs.org/api/child_process.html#child_process_child_process_execfilesync_file_args_options

I think we should follow the pattern here and consider calling it options.input

Reply:
Agreed, great suggestion

Reply:
Done